platform/core/src/ffi/java_interface.rs

```diff
@@ -202,14 +205,48 @@ impl core::PlatformHookRequest for BridgeToPlatform {
     }
 }
 
+#[no_mangle]
+pub extern "system" fn Java_coop_polypoly_core_JniApi_execFeatureRdfQuery(
+    env: JNIEnv,
+    _: JClass,
+    query: JString,
+    featureName: JString,
```

Comment:
There should be a separate API for configuring the `active_feature_id`, then this `featureName` will not be needed.